ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6605,4 +6605,18 @@ VMADC:
   - mixed_air_damper_percentage_command
   - mixed_air_damper_percentage_sensor
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+MAZTM:
+  description: "Zone temperature monitoring for MDF room with high & low temperature alarms, sensor failed alarm "
+  is_abstract: true
+  uses:
+  - high_limit_zone_air_temperature_setpoint
+  - low_limit_zone_air_temperature_setpoint
+  - zone_air_temperature_sensor
+  - high_zone_air_temperature_alarm
+  - low_zone_air_temperature_alarm
+  - failed_zone_air_temperature_alarm
+  - failed_alarm
```

Comment:
Don't need failed_alarm here. These should typically go on general types or on `SS`.